### ISSUE TITLE
regression: 0.5.0 fails to verify

### ISSUE THREAD

**Issue by nahuhh** [element-hq/element-x-android#3266]
### Steps to reproduce

1. Where are you starting? What can you see?
2. What do you click?
3. More steps…

with an existing account logged into element (not element x) for android
1. Download element x fdroid build
2. Login to x
3. Try to verify via emoji
4. It never connects. You never see the emoji screen, just says waiting for other device

element 0.4.15 fdroid verification works

#### What did you expect?
For it to show the emojis
#### What happened instead?
nothing

### Your phone model

Multiple

### Operating system version

Multiple

### Application version and app store

Element X version 0.5.0

### Homeserver

Multiple

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by licaon-kter** [comment]
There is no 0.5.0 in F-Droid, see https://github.com/element-hq/element-x-android/issues/3151 but only 0.4.15

**Comment by nahuhh** [comment]
The fdroid apk is available here:
https://github.com/element-hq/element-x-android/releases/tag/v0.5.0

i have not tried the standard apk @licaon-kter

**Comment by barathrm** [comment]
I seem to be getting this (or https://github.com/element-hq/element-x-android/issues/1189?) on 0.4.15 from f-droid.

I had somehow managed to verify and login before, but I logged out to debug another issue and now I can't verify against element desktop anymore.

1. Element X, "Confirm that it's you" screen. Tap "Use another device"
2. Element desktop: accept request, click "Compare unique emoji" start button
3. Element desktop now waits forever with a spinner 
4. Element X also spins forever saying "Waiting on other device..."

**Comment by xundeenergie** [comment]
The same with 0.6.0

**Comment by xundeenergie** [comment]
Please also have a look on this issue... 

https://github.com/element-hq/matrix-authentication-service/issues/3218

It fails with MAS, there are logfiles added.

**Comment by xundeenergie** [comment]
Since upgrade to v0.6.2 today, verification works with MAS.

**Comment by xundeenergie** [comment]
Again... verification is broken on v0.7.0

```
==> /var/log/matrix-synapse/homeserver.log <==                                                                                                                                                                     
                                                                                                                                                                                                                   
==> /var/log/nginx/matrix.schuerz.at.access.log <==                                                                                                                                                                
127.0.0.1 - - [11/Oct/2024:02:36:38 +0200] "POST /_matrix/client/unstable/org.matrix.simplified_msc3575/sync?pos=2714%2Fs1423894_90216077_4_2851108_235813_134_158077_6649543_0_37&timeout=30000 HTTP/1.1" 200 292 
"-" "Element X/0.7.0 (samsung SM-A536B; Android 14; UP1A.231005.007.A536BXXSBEXH3; Sdk 19b9a73ec)"                                                                                                                 
                                                                                                                                                                                                                   
==> /var/log/matrix-synapse/homeserver.log <==                                                                                                                                                                     
                                                                                                                                                                                                                   
==> /var/log/nginx/matrix.schuerz.at.access.log <==
127.0.0.1 - - [11/Oct/2024:02:36:42 +0200] "PUT /_matrix/client/v3/sendToDevice/m.key.verification.request/e23ef2fc61954570b9181b2a906c64d1 HTTP/1.1" 200 33 "-" "Element X/0.7.0 (samsung SM-A536B; Android 14; UP1A.231005.007.A536BXXSBEXH3; Sdk 19b9a73ec)"
127.0.0.1 - - [11/Oct/2024:02:36:42 +0200] "POST /_matrix/client/unstable/org.matrix.simplified_msc3575/sync?pos=0%2Fs1423892_90215949_0_2851102_235813_134_158078_6649460_0_37&timeout=30000 HTTP/1.1" 200 230 "-" "Element X/0.7.0 (samsung SM-A536B; Android 14; UP1A.231005.007.A536BXXSBEXH3; Sdk 19b9a73ec)"
```

But a verfication-request never reaches another device. 

Also there is no unverified session registered in MAS.
